include/tvm/target/tag.h

```diff
@@ -104,6 +104,12 @@ class TargetTagRegEntry {
    * \param config The config dict for target creation
    */
   inline TargetTagRegEntry& set_config(Map<String, ObjectRef> config);
+  /*!
+   * \brief Add a key-value pair to the config dict
+   * \param key The attribute name
+   * \param value The attribute value
+   */
+  inline TargetTagRegEntry& add_config(String key, ObjectRef value);
```

Comment:
`Add_config` seems to assume the key was not originally in the configuration, what about`with_config` which isn't related to whether the original config has the given key.

Reply:
Make sense. Will change the name into with_config.